src/dtos/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub(crate) mod record;
+
+pub(crate) trait ToDTO {
+    type OutputDTO;
+
+    fn to_dto(self) -> Self::OutputDTO;
+}
```

Comment:
And this trait is [`From`](https://doc.rust-lang.org/stable/std/convert/trait.From.html)